files/spark/bin/charmed-spark-history-server.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
maybe use `/bin/sh` instead of `/bin/bash`? You might need to reformat the variable `$SPARK_PROPERTIES_FILE` though